### PR TITLE
Lazy-loading support with lazysizes (https://github.com/aFarkas/lazysizes)

### PR DESCRIPTION
**Pull request by yuval-cloudinary**
Based on a work done for Swarovski

### CODE REVIEW

File: js/jquery.cloudinary.js [hunk 1]
```diff
@@ -4293,13 +4296,20 @@ var slice = [].slice,
             if (containerWidth !== 0) {
               switch (false) {
                 case !/w_auto:breakpoints/.test(dataSrc):
-                  requiredWidth = maxWidth(containerWidth, tag);
-                  dataSrc = dataSrc.replace(/w_auto:breakpoints([_0-9]*)(:[0-9]+)?/, "w_auto:breakpoints$1:" + requiredWidth);
+                  if (requiredWidth = maxWidth(containerWidth, tag)) {
```
Comment (patrick-tolosa) [inline js/jquery.cloudinary.js]: These changes seem unrelated the lazy loading (this solves a different bug if I remember correctly).

Can you please separate the lazyload solution to its own PR so we can understand the changes?

Comment (yuval-cloudinary) [reply js/jquery.cloudinary.js]: Thanks for catching this! I've removed the unrelated change

**Comment by patrick-tolosa** [comment]
Thanks for the PR,

Unfortunately, this PR currently conflicts with changes in master, more specifically it conflicts with the newly implemented solution for lazy loading.

Do you suggest we use this solution instead of the current implementation?
if so, could you explain what our current implementation doesn't cover?

before adding a dependency we'd like to make sure there's a good case for it.

**Comment by yuval-cloudinary** [comment]
@patrick-tolosa  At the time I've published my PR, there was no lazy loading support at all in this package. Looking at the newly merged implementation, I agree that both changes should not coincide.

Please notice that you should add the lazy loading to the JQuery plugin and the bundle, similar to this PR.
 Also, some documentation and author recognition should be added to the README
